app/src/main/java/org/xbmc/kore/ui/viewgroups/DynamicListView.java

```diff
@@ -167,6 +172,9 @@ public boolean isItemBeingDragged() {
     private AdapterView.OnItemLongClickListener mOnItemLongClickListener =
             new AdapterView.OnItemLongClickListener() {
                 public boolean onItemLongClick(AdapterView<?> arg0, View arg1, int position, long id) {
+                    if (!itemDraggingEnabled)
+                        return false;
+
                     mTotalOffset = 0;
 
                     mOriginalPosition = position;
```

Comment:
I haven't figured out why, but now dragging an item on the list will cause a npe in `onSwapFinished`

Reply:
I have not been able to reproduce the NPE. Can you be more specific in what circumstance this occurs?

Reply:
Forget about this, i can't reproduce it either, but i swear that i saw npes before...